### PR TITLE
[Logging] Added logback support

### PR DESCRIPTION
**Pull request by Tim2313**
Added logger to my project!!!

### CODE REVIEW

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <pattern>${PATTERN}</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="FILE"/>
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+</configuration>
```
Comment (Buga1chukY) [inline src/main/resources/logback.xml]: Always add new line in the end of the file.

File: src/main/java/org/example/MainServlet.java [hunk 2]
```diff
@@ -20,8 +23,8 @@ public class MainServlet extends HttpServlet {
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        logRequest(req);
 
+        LOGGER.info("We enter on path {}", req);
```
Comment (Buga1chukY) [inline src/main/java/org/example/MainServlet.java]: Does it work? Could you please provide the output? Is the output informative enough?

**Review by Buga1chukY — CHANGES_REQUESTED** [review]

File: src/main/resources/logback.xml (hunk 1 above)
Comment (Tim2313) [reply src/main/resources/logback.xml]: fixed

File: src/main/java/org/example/MainServlet.java (hunk 2 above)
Comment (Tim2313) [reply src/main/java/org/example/MainServlet.java]: Fixed

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -29,7 +29,8 @@ services:
       - 8080:8080
       - 5005:5005
     volumes:
-      - ../target/DeveloperApi.war:/usr/local/tomcat/webapps/DeveloperApi.war
+      - ../DeveloperApi/:/usr/local/tomcat/webapps/
```
Comment (MLGMAG) [inline docker/docker-compose.yml]: Maybe it is better to create a separate dir in the target folder and use it?

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/MainServlet.java [hunk 4]
```diff
@@ -24,7 +24,7 @@ public class MainServlet extends HttpServlet {
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
 
-        LOGGER.info("We enter on path {}", req);
+        LOGGER.info("We enter on path {}", req.getRequestURI());
```
Comment (Buga1chukY) [inline src/main/java/org/example/MainServlet.java]: Add string varieble before you add a 'req' to logger.info. Make string from httpServletRequest.

File: docker/docker-compose.yml (hunk 3 above)
Comment (Tim2313) [reply docker/docker-compose.yml]: fixed

File: src/main/java/org/example/MainServlet.java (hunk 4 above)
Comment (Tim2313) [reply src/main/java/org/example/MainServlet.java]: fixed

File: docker/docker-compose.yml (hunk 3 above)
Comment (MLGMAG) [reply docker/docker-compose.yml]: Good solution! I like it!

**Review by MLGMAG — APPROVED** [review]

**Pull request merged**